core/lib/basic_types/src/pubdata_da.rs

```diff
@@ -1,15 +1,17 @@
+//! Types related to data availability.
+
 use chrono::{DateTime, Utc};
 use num_enum::TryFromPrimitive;
 use serde::{Deserialize, Serialize};
-use zksync_basic_types::L1BatchNumber;
-use zksync_config::configs::eth_sender::PubdataSendingMode;
+
+use crate::L1BatchNumber;
 
 /// Enum holding the current values used for DA Layers.
 #[repr(u8)]
-#[derive(Debug, Clone, Copy, Deserialize, PartialEq, Serialize)]
-#[derive(TryFromPrimitive)]
-pub enum PubdataDA {
+#[derive(Debug, Clone, Copy, Default, PartialEq, Deserialize, Serialize, TryFromPrimitive)]
+pub enum PubdataSendingMode {
```

Comment:
This refactoring assumes that `PubdataSendingMode` and `PubdataDA` in fact refer to the same concept. I'm not 100% sure that is the case. The current definitions are obviously identical, but maybe they'll diverge in the future?

Reply:
my guess was that these were duplicated as a workaround to some circular dependency, so they shouldn't diverge in the future, but maybe I'm missing some other reason